src/components/ListItem.js

```diff
@@ -10,6 +10,17 @@ const ListItem = ({ item, token }) => {
   const [isPurchased, setPurchased] = useState(false);
   let numberOfPurchases = item.number_purchases || 0;
   let hoursDiff = getDifferenceInHours(item.last_purchased);
+  let daysDiff = getDifferenceInDays(item.last_purchased);
+
+  let className = '';
+  className =
```

Comment:
noticed you're declaring className, and then immediately assigning the value here.